R/derive_vars_aage.R

```diff
@@ -93,10 +100,18 @@ derive_vars_aage <- function(dataset,
   start_date <- assert_symbol(enexpr(start_date))
   end_date <- assert_symbol(enexpr(end_date))
   assert_data_frame(dataset, required_vars = expr_c(start_date, end_date))
-  assert_character_scalar(
-    age_unit,
-    values = c("years", "months", "weeks", "days", "hours", "minutes", "seconds")
-  )
+
+  age_unit <- get_unified_time_unit(age_unit)
```

Comment:
This should be removed (and `case_sensitive = FALSE` be used in the `assert_character_scalar()` call). Otherwise the unit variable is not set correctly. Consider for example
```
> derive_vars_aage(data, age_unit = "wks")
# A tibble: 1 × 4
  BRTHDT     RANDDT      AAGE AAGEU
  <date>     <date>     <dbl> <chr>
1 1984-09-06 2020-02-24  1850 weeks
```
`AAGEU` should be `"wks"`.